etest/docker/container.py

```diff
@@ -24,27 +24,30 @@ def commit(container: Container, tag: str, repository: str, *args, **kwargs):
     return repo + ":" + tag
 
 
-def create(overlay: str, *args, **kwargs):
+def create(overlay="", autoconf=True, *args, **kwargs):
```

Comment:
Type hint was lost here.  Using boolean parameters typically indicates an interface issue but that's not clear yet.